sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,233 @@
+ - Added [InvokeSIMD](doc/extensions/InvokeSIMD/InvokeSIMD.asciidoc) and
+   [Uniform](doc/extensions/Uniform/Uniform.asciidoc) extensions [72e1611]
+ - Added [Matrix Programming Extension for DPC++ document](doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc) [ace4c733]
+ - Implemented SYCL 2020 `sycl::span` [9356d53]
+ - Added [device-if](doc/extensions/DeviceIf/device_if.asciidoc) extension
+   [4fb95fc]
+ - Added a [programming guide](doc/MultiTileCardWithLevelZero.md) for
+   multi-tile and multi-card under Level Zero backend [d581178a]
+
+## Improvements
+### SYCL Compiler
+ - Use `opencl-aot` instead of `aoc` when AOT flow for FPGA Emulator is
+   triggered [3a99558]
+ - Allowed for using an external host compiler [6f0ad1a]
+ - Cleaned up preprocessing output when `-fsycl` option is passed [3a18db6]
+ - Allowed kernel names in anonymous namespace [e47dbad]
+ - Set default value of `-sycl-std` to `2020` when `-fsycl-is-device` or
+   `-fsycl-is-host` specified [680adc0]
+ - Added implication of `-fPIC` compilation for wrapped object when using
+   `-shared` [1754934]
+ - Added a diagnostic for `-fsycl` and `-ffreestanding` as non-supported
+   combination [a36c6720]
+ - [ESIMD] Renamed `simd::format` to `simd::bit_cast_view` [653dede1]
+ - Allowed `[[sycl::work_group_size_hint]]` to accept constant expr args
+   [ef8e4019]
+ - Deprecated the old-style SYCL attributes according to the SYCL 2020 spec
+   [001bbd42]
+ - Deprecated `[[intel::reqd_work_group_size]]` attribute spelling, please use
+   `[[sycl::reqd_work_group_size]]` instead [8ef7eacc]
+ - Enabled native FP atomics by default. Defining the
+   `SYCL_USE_NATIVE_FP_ATOMICS` macro explicitly is no longer required - it is
+   now automatically defined for hardware targets with "native" support for
+   atomic functions. [0bbb68ee]
+ - Switched to ignoring `-O0` option for device code when compiling for FPGA
+   with hardware [7d94edf4]
+ - Allowed for known aliases to be used for `-fsycl-targets`. Passing
+   `*-unknown-unknown-sycldevice` components of the SYCL target triple is no
+   longer necessary. [9778952a]
+ - [ESIMD] Added support for half type in ESIMD intrinsics [d5958ebf]
+ - Implemented `sycl::kernel::get_kernel_bundle` method [69a68a6d]
+ - Added a diagnostic in case of timing issues for FPGA AOT [c69a3115]
+ - Added support for C `memcpy` usages in the device code [76051ccf]
+ - [ESIMD] Added support for vectorizing scalar function [3fc66cc]
+ - Disabled vectorization and loop transformation passes because loop unrolling
+   in "SYCL optimization mode" used default heuristic, which is tuned the code
+   for CPU and might not have been profitable for other devices [ff6929e6]
+### SYCL Library
+ - Added an exception throw if no matched device is found when
+   `SYCL_DEVICE_FILTER` is set regardless of `device_selector` used [ef4e6dd]
+ - Changed event status update to complete without waiting when run on CUDA
+   devices [be7c1cb]
+ - Added `sycl::bit_cast` [d4b66bd]
```

Comment:
Isn't it a new feature?

Reply:
Sometimes it's hard to say whether a change is a feature or an improvement. This is very simple one and has been supported in `detail` namespace for some time already, that's why I put it into improvements section.
Please, let me know if you prefer having it in the `new features` section.

Reply:
Strictly speaking, `detail` was not intended for users, so for them it is more like a new feature, I guess.